src/test/java/jalau/at18/katas/sarai/banck/userstoryTest.java

```diff
@@ -0,0 +1,28 @@
+package jalau.at18.katas.sarai.banck;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import jalau.at18.katas.bankocr.sarai.Userstory1;
+
+public class userstoryTest {
```

Comment:
Please always use Pascal case for class names (first letter in capital case)

Reply:
Fix class names